Bugfix-June

katsuragiyy opened the pull request:
Patch notes for the June update --> https://github.com/deliciousmods/1956_beta/wiki/June-2025-Patch-Notes

If you have checked a country's focus tree for bugs, then add it to the list of checked-countries in this document. Adding a date of when the country was last checked is also heavily recommended. Link to the document --> https://docs.google.com/document/d/1w2dStPoE_LBmcskbwXRJ5pKvNAj06W3awRq6e_ssPP4/edit?usp=sharing

katsuragiyy committed:
Typo in LIE

TheBrit7 committed:
Make Dominican republic fascist

TheBrit7 committed:
Update DOM.txt

Commit:
Old RT56 Germany Icons are Back!

Well actually some of them...

Commit:
Added tooltip

Commit:
Estonia core fix

TheBrit7 committed:
Trait

TheBrit7 committed:
Adjustments

BiccyThiccy committed:
Removing typo in Norton de Matos's description

Commit:
Germany adjustments

BiccyThiccy committed:
Merge branch 'Beta-Edits' into bugfixing-june

katsuragiyy committed:
Johor

marijn211 committed:
Fixed 2 bugs+1 adjustment in ROM tree

marijn211 committed:
Fix AUS event mixup

marijn211 committed:
Bugfix from Reddit

marijn211 committed:
Clean up Turkey AI plans

marijn211 committed:
Fix incorrect focuses in Ottoman AI plan

marijn211 committed:
Typo

marijn211 committed:
Baltic Countries Ultimatum sent just once, corrected loc

Also added a bypass effect to the USSR Bessarabia focus

Commit:
Siberian Railways LLC

Commit:
Update soviet_old.txt

katsuragiyy committed:
DOM flag fix

katsuragiyy committed:
brunei colour fix

katsuragiyy committed:
Got rid of Muscat and Oman raids

Just integrate them through the lower arabia thing at this point..

katsuragiyy committed:
HOL integrate decision

BiccyThiccy committed:
Many ROM Fixes and Improvements

- Added available_if_capitulated = yes to a few focus's where it seems logical (why not?)
- Changed create_faction = "" into strings, and removed 'the' from faction names to match vanilla. This is so translation submods can modify them without overwriting our entire focus tree file.
- Romania should no longer pull countries from faction via invites anymore, but it's possible I missed some. Only found 2 instances where this could happen.
- Many sanity checks were adjusted or added.
-All Wargoal AI was adjusted -- they will hopefully no longer get stuck in stupid/crash wars.
- Added some line-skips to some ugly ah effects

BiccyThiccy committed:
ROM - Soviets breaking Molotov-Ribbentrop no longer ruins historical tree

BiccyThiccy committed:
Making content scoping for SOV focus's work on old SOV tree + non-dlc vanilla branch

@marijn2112 If you have time I'd appreciate a lookover.

Error log looks good and I diff-checked it, but you know the tree a lot better than me 😛

BiccyThiccy committed:
CHI - Fix German Intervention Path

BiccyThiccy committed:
Incorporating vanilla changes to SOV Secure Leningrad

BiccyThiccy committed:
Porting missing TOA LOC

BiccyThiccy committed:
Undoing ENG getting pulled from the EU formable

BiccyThiccy committed:
TUR - Hopefully fixing Ottomans randomly pulling GER from Mitteleuropa

BiccyThiccy committed:
PHI - Changing wargoal AI + minor polish

Commit:
error.log cleanup

BiccyThiccy committed:
INS - Non-DLC user fixes + Improving Wargoal AI

"My child launches properly tested countries that the AI will definitely will not declare on the entire world with" 😇🪽

😈

BiccyThiccy committed:
Fixing 2 errors for those with no AAT

BiccyThiccy committed:
Fixing one no-AAT error

BiccyThiccy committed:
POL should no longer pull countries from factions + code cleanup

BiccyThiccy committed:
Bag of bugfixes

- Simplified some R56 RAJ code
- Fixed two BEL error log spammers from character typo
- Fixed RAJ Azad hind civil war not giving manpower idea
- Fixed minor typo in a tooltip on Brazil

BiccyThiccy committed:
Squashing 22 RAJ errors by importing advisors

BiccyThiccy committed:
Fixing a couple IRQ errors from different-named characters

BiccyThiccy committed:
A couple bug/spell-fixes + lots of wargoal polishing

- Minor bugs/spell-fixes that would be too lengthy to list
- Added many missing will_lead_to_war_with's
- Many better AI conditions for completing wargoal-related focuses (hopefully to reduce AI stupidity and game-ruining).

BiccyThiccy committed:
FRA - Giving all wargoal focuses AI factors for army size / surrender-progress / wars

BiccyThiccy committed:
Merge branch 'Beta-Edits' into bugfixing-june

BiccyThiccy committed:
GRE - Importing vanilla stuff + couple fixes/improvements

- Made file more similar to vanilla (PR checkers nightmare :devil)
- Imported new focuses added in recent updates
- Imported recent balance changes from Paradox
- Fixed a few bugs
- On focuses that gave wargoals, did a check for will_lead_to_war_with and proper AI, fixed accordingly.
- Added focus autohiding/offsetting support

BiccyThiccy committed:
Fixing more stupid wargoal AI + other polish/fixes

BiccyThiccy committed:
More will_lead_to_war_with where applicable, other polish

Commit:
Finland now can't lay mines in **B R A Z I L ! ! !**

Commit:
Added the "new" (like two years old) warlord rework in it's full beauty

Commit:
I've decided to rename Venezuela to America - Donald John Trump, 1984

BiccyThiccy committed:
PAR - adding missing LOC, will_lead_to_war_with and ai war triggers,

BiccyThiccy reviewed (commented):
^Forgot to say, my Paraguay commit also makes the file more similar to vanilla which explains the large difff

Commit:
Vargas RT56 portrait with TOA

Commit:
sanity repair

BiccyThiccy committed:
POL - adding will_lead_to_war_with + improving wargoal focus AI

BiccyThiccy committed:
NOR - sanity for NOR_baltic_anti-communist_league path

marijn211 committed:
Merge branch 'Beta-Edits' into bugfixing-june

Commit:
fordlandia fix

Commit:
Update USA_organization.txt

Improved comments, compared with vanilla (WinMerge)

Commit:
Update ethiopia.txt

Checked by WinMerge

Commit:
GoE Afghanistan Tree WinMerged

Commit:
Vanilla AFG Fixes

Commit:
Update AFG - Afghanistan.txt

BiccyThiccy committed:
Importing a couple missing tooltips

Commit:
AFG fixes, enabled focus tree switching

At this point GoE AFG is bug free
TODO check a few advisors

Commit:
Saadabad progress

Commit:
AFG adjustments

Co-Authored-By: Dutch <128647164+DutchPlanDerLindeSus@users.noreply.github.com>

Commit:
Saadabad branch for all

BiccyThiccy committed:
Amend commit "Saadabad branch for all"

BiccyThiccy committed:
DEN - adding will_lead_to_war_with (some ported from van, some added by me)

TheBrit7 committed:
Jungles in Australia

Commit:
Update 00_static_modifiers.txt

WinMerge

Commit:
Update 00_intelligence_agencies.txt

Commit:
Update CZE_organization.txt

WinMerged

Commit:
Ethiopia Fixes

WinMerged & Imrpoved Balance

Tooltips as well :)

marijn211 committed:
Merge branch 'Beta-Edits' into bugfixing-june

BiccyThiccy reviewed (approved):
No one but Marijn merge plz

BiccyThiccy committed:
Merge branch 'Beta-Edits' into bugfixing-june

marijn211 commented on common/ai_strategy_plans/SWE_r56_alternate_strategy_plan.txt:
```diff
@@ -1311,7 +1332,10 @@ SWE_r56_nordic_union_plan = {
-			has_completed_focus = SOV_the_centre
+			OR = {
+				has_completed_focus = SOV_the_centre
+				NOT = { has_completed_focus = SOV_form_a_fifth_column_old }
```
This will always be true if the DLC tree is used, instead of `NOT = { has_completed_focus = SOV_form_a_fifth_column_old }`, I think checking for the old tree's great purge is probably best

The other checks for the old soviet tree seems excellent

Xaelorichero-02 replied:
![image](https://github.com/user-attachments/assets/9c1e2070-9975-464e-8d74-1d63f55c6ee5)
@marijn2112 Like this?

marijn211 replied:
@Xaelorichero-02 the goal here is to see whether SOV is going historical, so there should be no NOT.
Also, Biccy astutely noted that the great purge focus might be a bit long, so checking focus progress on the great purge focus might be better

`focus_progress = {
  focus = SOV_great_purge
  progress > 0.1
}`

DutchPlanDerLindeSus committed:
Update r56_afghanistan.txt

marijn211 commented on common/national_focus/uruguay.txt:
```diff
@@ -1602,9 +1602,16 @@ focus_tree = {
 		relative_position_id = URG_colorado_victory
+		ai_will_do = {
+			base = 0
+			modifier = {
```
This focus now has 2 ai_will_do blocks, please merge them

Xaelorichero-02 replied:
Fixed

Commit:
uruguay fix

BiccyThiccy commented:
@Xaelorichero-02 Can't use a factor when the base is 0 because it multiplies it.

I'll fix it real quick, this is my mistake anyway

marijn211 commented on common/national_focus/china_warlord.txt:
```diff
@@ -1390,4 +1390,136 @@ focus_tree = {
+		completion_reward = {
+			IF = {
+				limit = {
+					has_idea = CHI_warlord_state_idea
```
This change should've been on a content PR and not the bugfix branch

Xaelorichero-02 replied:
This doesn't really... The idea doesn't modify the AI but I may see the name conflict

marijn211 replied:
@Xaelorichero-02 I made a mistake, I corrected my message

BiccyThiccy committed:
Update uruguay.txt @marijn2112

Commit:
quick fix

BiccyThiccy committed:
Update SWE_r56_alternate_strategy_plan.txt

TheBrit7 committed:
Merge branch 'Beta-Edits' into bugfixing-june

TheBrit7 committed:
Update mongolia.txt

TheBrit7 committed:
Delete Portrait_Germany_Konrad_Adenauer.dds

BiccyThiccy committed:
@TheBrit7 Amending previous commit

I don't like WIP content sitting there but I don't think you're going about this the right way, doesn't make things any less
 cluttered for a Mongol player after all.

I just changed it so the right branches hide if one of those first two focuses are selected

BiccyThiccy committed:
AI shouldn't do MON unfinished paths anymore

Commit:
Fixed broken GRE factions for new advisors

Commit:
Changed Grigore Grafencu's trait

Commit:
Merge branch 'Beta-Edits' into bugfixing-june

Commit:
Merge branch 'Beta-Edits' into bugfixing-june

marijn211 committed:
Fix character mistake AFG
